tests/contrib/BUILD

```diff
@@ -165,3 +166,26 @@ docker_push(
     "kaniko_latest",
     "kaniko_debug",
 ]]
+
+container_repro_test(
+    name = "set_cmd_repro_test",
+    base = "@bazel_latest//image",
```

Comment:
dont pass a base explicitly in these tests

Reply:
As discussed offline, this is needed for the container_image impl function to be able to use the image intermediate format when mounting the project srcs into the Bazel image. Passing the `base` differently requires passing in the image tar file, which affects performance significantly.